lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1835,6 +1835,11 @@ def test_xmodule_runtime_publish_delete(self):
     def test_score_change_signal(self, send_mock):
         """Test that a Django signal is generated when a score changes"""
         self.set_module_grade_using_publish(self.grade_dict)
+        user_module = StudentModule.objects.get(
+            student_id=self.student_user.id,
+            module_state_key=self.problem.location,
+            course_id=self.course.location.course_key
+        )
```

Comment:
I'm a little bit confused why we need to access `user_module` here - is this still needed?